<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MasterProject</class>
 <widget class="QMainWindow" name="MasterProject">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>521</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>440</width>
    <height>790</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>231</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Ribbons</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>231</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>Sheets</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="monolayer_ribbon">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>191</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Monolayer</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bilayer_ribbon">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>50</y>
      <width>191</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Bilayer</string>
    </property>
   </widget>
   <widget class="QPushButton" name="twistedbilayer_ribbon">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>140</y>
      <width>191</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Twisted Bilayer</string>
    </property>
   </widget>
   <widget class="QPushButton" name="trilayer_ribbon">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>191</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Trilayer</string>
    </property>
   </widget>
   <widget class="QPushButton" name="twistedbilayer_sheet">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>360</y>
      <width>191</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Twisted Bilayer</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bilayer_sheet">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>270</y>
      <width>191</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Bilayer</string>
    </property>
   </widget>
   <widget class="QPushButton" name="monolayer_sheet">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>191</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Monolayer</string>
    </property>
   </widget>
   <widget class="QPushButton" name="trilayer_sheet">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>360</y>
      <width>191</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Trilayer</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>440</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Version 1.0.</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>440</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMaster_Project">
    <property name="title">
     <string>Master Project</string>
    </property>
   </widget>
   <addaction name="menuMaster_Project"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
